### ISSUE TITLE
create_env lbzip2 missing

### ISSUE THREAD

**Issue by c1au6i0** [luciorq/condathis#14]
running `condathis` in a `rocker/tidyverse` container.
```
> condathis::create_env(
+         c("curl==7.78.0", "libcurl", "openssl"),
+         env_name = "openssl-linux-env",
+          verbose = FALSE
+      )
trying URL 'https://github.com/mamba-org/micromamba-releases/releases/download/2.0.2-1/micromamba-linux-64.tar.bz2'
Content type 'application/octet-stream' length 6028157 bytes (5.7 MB)
==================================================
downloaded 5.7 MB

tar (child): lbzip2: Cannot exec: No such file or directory
tar (child): Error is not recoverable: exiting now
/usr/bin/tar: Child returned status 2
/usr/bin/tar: Error is not recoverable: exiting now
Error in `install_micromamba()`:
✖ /home/rstudio/.local/share/condathis/micromamba/bin/micromamba was not extracted succesfully.
! This error can caused by missing `bzip2` library.
Run `rlang::last_trace()` to see where the error occurred.
Warning messages:
1: In base::normalizePath(umamba_bin_path) :
  path[1]="/home/rstudio/.local/share/condathis/micromamba/bin/micromamba": No such file or directory
2: In utils::untar(tarfile = full_dl_path, exdir = fs::path_expand(untar_dir)) :
  ‘/usr/bin/tar -xf '/home/rstudio/.local/share/condathis/micromamba-dl.tar.bz2' -C '/home/rstudio/.local/share/condathis/micromamba'’ returned error code 2
```

**Comment by luciorq** [comment]
Related to #10

**Comment by luciorq** [comment]
Failover method implemented in: 0b4f562

Thank you!
